functions/go/apply-replacements/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.17-alpine3.13
+ENV CGO_ENABLED=0
+WORKDIR /go/src/
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN go build -o /usr/local/bin/function ./
+
+#############################################
+
+FROM alpine:3.13
+COPY --from=0 /usr/local/bin/function /usr/local/bin/function
+ENTRYPOINT ["function"]
```

Comment:
This Dockerfile is not needed because functions under `./functions/go/` should use the same Dockerfile defined in https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/build/docker/go/Dockerfile

Reply:
+1

Reply:
done